prometheus/exporter/watchdog.py

```diff
@@ -125,12 +125,18 @@ def ssh_exec(host_config, command, histogram=ssh_histogram):
         port = 22
         if "sshport" in host_config:
             port = int(host_config["sshport"])
+        key_filename = None
```

Comment:
Well, watchdog have different runtime environment compare to paictl.py: watchdog is running in container and pai-master node. The key-filename exist in host executing paictl.py may not necessary exist in watchdog container.

You should maybe copy the key-filename into configMap like `pai-management/bootstrap/watchdog/watchdog-configmap.yaml.template` and then mount into watchdog container

Reply:
Thanks for the information, I will try your suggestions.

Reply:
use kubernetes secret to achieve this.